common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/FileUtil.java

```diff
@@ -436,6 +436,63 @@ public static String readContents( InputStream contents ) throws IOException
         return lines.toArray( new String[lines.size()] );
     }
 
+    public static String[] readLinesFromFile( File file, boolean includeNewlines  )
```

Comment:
Just add this argument to the previous method instead of copying and making a whole new method but add another method that doesn't have the boolean and restore the previous method behavior.